app/controllers/static_pages_controller.rb

```diff
@@ -1,5 +1,10 @@
 class StaticPagesController < ApplicationController
-  def home; end
+  def home 
+    return if logged_in?
+
+    @micropost = current_user.microposts.build
+    @feed_items = current_user.feed.paginate(page: params[:page]).per `number`
```

Comment:
```suggestion
    @feed_items = current_user.feed.paginate(page: params[:page]).per 5
```
Don't forget to move 5 to setting.yml file and call it here

Reply:
done